GoogleDataTransport/GDTWatchOSTestApp/GDTWatchOSTestApp.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,705 @@
+		9505C33A2405B30F00534CF2 /* InterfaceController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9505C3392405B30F00534CF2 /* InterfaceController.swift */; };
+		9505C33C2405B30F00534CF2 /* ExtensionDelegate.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9505C33B2405B30F00534CF2 /* ExtensionDelegate.swift */; };
+		9505C33E2405B30F00534CF2 /* NotificationController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9505C33D2405B30F00534CF2 /* NotificationController.swift */; };
+		9505C3402405B31000534CF2 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 9505C33F2405B31000534CF2 /* Assets.xcassets */; };
+		9505C3512405F0C500534CF2 /* gdthelper.swift in Sources */ = {isa = PBXBuildFile; fileRef = 9505C3502405F0C500534CF2 /* gdthelper.swift */; };
+		BA91CC6245B2B5DAA79D5F52 /* Pods_GDTWatchOSTestAppWatchKitApp.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 398CC1DDCF216D1E3868D054 /* Pods_GDTWatchOSTestAppWatchKitApp.framework */; };
+/* End PBXBuildFile section */
+
+/* Begin PBXContainerItemProxy section */
+		9505C3272405B30E00534CF2 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 9505C31B2405B30D00534CF2 /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = 9505C3242405B30E00534CF2;
+			remoteInfo = "GDTWatchOSTestApp WatchKit App";
+		};
+		9505C3362405B30F00534CF2 /* PBXContainerItemProxy */ = {
+			isa = PBXContainerItemProxy;
+			containerPortal = 9505C31B2405B30D00534CF2 /* Project object */;
+			proxyType = 1;
+			remoteGlobalIDString = 9505C3332405B30F00534CF2;
+			remoteInfo = "GDTWatchOSTestApp WatchKit Extension";
+		};
+/* End PBXContainerItemProxy section */
+
+/* Begin PBXCopyFilesBuildPhase section */
+		9505C3482405B31000534CF2 /* Embed App Extensions */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 13;
+			files = (
+				9505C3352405B30F00534CF2 /* GDTWatchOSTestAppWatchKitExtension.appex in Embed App Extensions */,
+			);
+			name = "Embed App Extensions";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		9505C34C2405B31000534CF2 /* Embed Watch Content */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "$(CONTENTS_FOLDER_PATH)/Watch";
+			dstSubfolderSpec = 16;
+			files = (
+				9505C3262405B30E00534CF2 /* GDTWatchOSTestAppWatchKitApp.app in Embed Watch Content */,
+			);
+			name = "Embed Watch Content";
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+/* End PBXCopyFilesBuildPhase section */
+
+/* Begin PBXFileReference section */
+		12A683F830B22F5108F84577 /* Pods-GDTWatchOSTestAppWatchKitExtension.debug.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-GDTWatchOSTestAppWatchKitExtension.debug.xcconfig"; path = "Target Support Files/Pods-GDTWatchOSTestAppWatchKitExtension/Pods-GDTWatchOSTestAppWatchKitExtension.debug.xcconfig"; sourceTree = "<group>"; };
```

Comment:
make sure you run `pod deintegrate` before committing so project.pbxproj is clean without installing pods information.